R/description.R

```diff
@@ -193,16 +194,16 @@ use_description_field <- function(name, value, overwrite = FALSE) {
   }
 
   ui_done("Setting {ui_field(name)} field in DESCRIPTION to {ui_value(value)}")
-  desc::desc_set(name, value, file = proj_get())
+  desc$set(name, value)
+  desc$write()
+
   invisible()
 }
 
 use_description_list <- function(key,
                                  values,
-                                 append = TRUE,
-                                 desc = NULL) {
-  desc_provided <- !is.null(desc)
-  desc <- desc %||% desc::desc(file = proj_get())
+                                 append = TRUE) {
+  desc <- desc %||% proj_desc()
```

Comment:
`desc` still needs to be defined somewhere.

Reply:
Oops